comfy/model_management.py

```diff
@@ -4,6 +4,7 @@
 LOW_VRAM = 2
 NORMAL_VRAM = 3
 HIGH_VRAM = 4
+MPS = 4
```

Comment:
should be 5?

Reply:
Yeah, typo. Sorry, I'll fix it as soon as I get to the computer.